app/src/main/java/com/gazlaws/codeboard/CodeBoardIME.java

```diff
@@ -186,6 +192,9 @@ public void onKey(int primaryCode, int[] KeyCodes) {
                         ke = KeyEvent.KEYCODE_ESCAPE;
                         break;
                     case 32:
+                        /* Added by Dimitris Stoukogiannis (aka ShadyGeek) on 14/2/2023.
+                            Boolean variable is set to false, once the space button is pressed by the user */
```

Comment:
Nit: No need of this comment either.

Reply:
Thanks for the feedback man, you've been so helpful, thank you! Shall I correct it and make a pull request again?

Reply:
Yes please.
I remember my college days haha! Hope you get a good score 🚀

Reply:
Hahaha thank you for your time, it means a lot. To save you some, I will now go on to make the changes you mentioned and submit the pull request again. Regarding the comments, do you want me to keep the one you liked?

Reply:
Yes keep that one - it's useful